bundles/company-users-bulk-rest-api/src/FondOfOryx/Zed/CompanyUsersBulkRestApi/Business/Manager/BulkManager.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+namespace FondOfOryx\Zed\CompanyUsersBulkRestApi\Business\Manager;
+
+use ArrayObject;
+use FondOfOryx\Shared\CompanyUsersBulkRestApi\CompanyUsersBulkRestApiConstants;
+use FondOfOryx\Zed\CompanyUsersBulkRestApi\Business\Permission\PermissionCheckerInterface;
+use FondOfOryx\Zed\CompanyUsersBulkRestApi\Business\PluginExecutioner\BulkDataEnrichmentPluginExecutionerInterface;
+use FondOfOryx\Zed\CompanyUsersBulkRestApi\Dependency\Facade\CompanyUsersBulkRestApiToEventFacadeInterface;
+use Generated\Shared\Transfer\CompanyUsersBulkPreparationTransfer;
+use Generated\Shared\Transfer\CompanyUserTransfer;
+use Generated\Shared\Transfer\RestCompanyUsersBulkItemCollectionTransfer;
+use Generated\Shared\Transfer\RestCompanyUsersBulkRequestTransfer;
+use Generated\Shared\Transfer\RestCompanyUsersBulkResponseTransfer;
+
+class BulkManager implements BulkManagerInterface
+{
+    protected PermissionCheckerInterface $permissionChecker;
+
+    /**
+     * @var \FondOfOryx\Zed\CompanyUsersBulkRestApi\Dependency\Facade\CompanyUsersBulkRestApiToEventFacadeInterface
+     */
+    protected CompanyUsersBulkRestApiToEventFacadeInterface $eventFacade;
+
+    /**
+     * @var \FondOfOryx\Zed\CompanyUsersBulkRestApi\Business\PluginExecutioner\BulkDataEnrichmentPluginExecutionerInterface
+     */
+    protected BulkDataEnrichmentPluginExecutionerInterface $enrichmentPluginExecutioner;
+
+    /**
+     * @param \FondOfOryx\Zed\CompanyUsersBulkRestApi\Business\Permission\PermissionCheckerInterface $permissionChecker
+     * @param \FondOfOryx\Zed\CompanyUsersBulkRestApi\Dependency\Facade\CompanyUsersBulkRestApiToEventFacadeInterface $eventFacade
+     */
+    public function __construct(
+        PermissionCheckerInterface                    $permissionChecker,
+        CompanyUsersBulkRestApiToEventFacadeInterface $eventFacade,
+        BulkDataEnrichmentPluginExecutionerInterface  $enrichmentPluginExecutioner
+    )
+    {
+        $this->permissionChecker = $permissionChecker;
+        $this->eventFacade = $eventFacade;
+        $this->enrichmentPluginExecutioner = $enrichmentPluginExecutioner;
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\RestCompanyUsersBulkRequestTransfer $restCompanyUsersBulkRequestTransfer
+     * @return \Generated\Shared\Transfer\RestCompanyUsersBulkResponseTransfer
+     */
+    public function handleBulkRequest(
+        RestCompanyUsersBulkRequestTransfer $restCompanyUsersBulkRequestTransfer
+    ): RestCompanyUsersBulkResponseTransfer
+    {
+//        if (!$this->permissionChecker->check($restCompanyUsersBulkRequestTransfer)) {
+//            return $this->createEmptyResponseTransfer()
+//                ->setCode(CompanyUsersBulkRestApiConstants::ERROR_CODE_PERMISSION_DENIED)
+//                ->setError(CompanyUsersBulkRestApiConstants::ERROR_MESSAGE_MISSING_PERMISSION);
+//        }
```

Comment:
Remove unused code.

Reply:
Its not unused, only uncommented for testing without permission check :)